.github/workflows/testdriver.yml

```
@@ -0,0 +1,38 @@
+            echo "creating .env file and setting OPENAI_API_KEY"
+            Set-Content -Path ".env" -Value "OPENAI_API_KEY=${{env.OPENAI_API_KEY}}"
+            echo "starting dev server"
+            Start-Process -NoNewWindow -PassThru "npm" -ArgumentList "run", "dev"
+            echo "starting chrome"
+            Start-Process "C:/Program Files/Google/Chrome/Application/chrome.exe" -ArgumentList "--start-maximized", "--load-extension=$(pwd)/node_modules/dashcam-chrome/build", "${{env.LOCALHOST_URL}}"
+            exit
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          FORCE_COLOR: "3"
+          OPENAI_API_KEY: ${{ secrets.OPENAI_API_KEY }}
+          LOCALHOST_URL: "http://localhost:3000"
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)